src/Dashboard/View.purs

```diff
@@ -56,6 +57,18 @@ formatStatus { id: Gitlab.PipelineId id, status } =
     , H.text $ show status
     ]
 
+formatName :: ∀ p a. Model.NameRow -> HTML p a
+formatName  { id
+            , name: Gitlab.ProjectName name
+            , name_with_namespace: Gitlab.ProjectNameWithNamespace namespace
+            }
+            =
+  H.div []
+        [ H.text $ fromMaybe "" (head $ split (Pattern "/") namespace)
```

Comment:
Maybe a `$` between `fromMaybe ""` and `head` instead of parenthesis (for consistency).